.agents/skills/populate-jtbds/SKILL.md

```
@@ -121,22 +125,79 @@ Note: the `object` value may be a Heimdall template like
 ```
-GET https://lfx-api.dev.v2.cluster.linuxfound.info<openapi3.yaml path>
+GET https://lfx-api.dev.v2.cluster.linuxfound.info<openapi3.json path>
 ```
 
 **GitHub fallback (for any that return non-200):**
 
-Search the `linuxfoundation` GitHub organization for `openapi3.yaml` using the
+Search the `linuxfoundation` GitHub organization for `openapi3.json` using the
 RuleSet `metadata.name` as the repo name hint:
 
 ```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)